src/lib/Configuration/AggregateProvider.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformRichText\Configuration;
+
+use EzSystems\EzPlatformRichText\API\Configuration\ProviderService;
+
+/**
+ * RichText configuration provider, providing configuration by aggregating different sources.
+ *
+ * @internal type-hint \EzSystems\EzPlatformRichText\API\Configuration\ProviderService
```

Comment:
@adamwojs this one is intentional. We use the phrase "for internal use..." if there's no better explanation. However here most common mistake would be to refer to the implementation instead of API interface. This doc explains to a developer how to resolve that (you can see it if you hover over struck-through definition in your code).